### PR TITLE
docs: modify demo image in intermediate-inputs.md

### PR DESCRIPTION
**Pull request by synoti21**
### Motivation
스크린샷 2023-09-06 오후 3 25 39
스크린샷 2023-09-06 오후 3 25 44

In "Intermediate Parameters" page in argoproj.github.io, there are two clickable images that users cannot know that it can play demo videos. 

### Modifications

I replaced them with two images that have YouTube logo that users can easily know it's a playable demo video. I considered to replace with embed videos, but since another contributor changed to image with YouTube logo in other documents, I also changed in the same way for consistency.

intermediate-inputs

### CODE REVIEW

File: docs/assets/intermediate-inputs.png
Comment (agilgur5) [inline docs/assets/intermediate-inputs.png:1]: Could you crop this image to not show your bookmarks, URL bar, tabs, etc?

**Comment by agilgur5** [comment]
Similar to #11732

File: docs/assets/intermediate-inputs.png
Comment (synoti21) [reply docs/assets/intermediate-inputs.png:1]: Okay, I'll crop it right now

Comment (agilgur5) [reply docs/assets/intermediate-inputs.png:1]: Oh my bad, I didn't realize the top bar is actually _from the video itself_, not a screenshot you made yourself.

Crop is still good to have though, thanks!

**Review by agilgur5 — APPROVED** [review]

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**